### PR TITLE
Decimals: replace hard-coded token decimals (6) by constant

### PR DESCRIPTION
**Pull request by Evalir**
** Merge after #33 **

Little addition—changes all hard-coded token decimal values for wPOKT to a constant. This is for more clarity, code-wise. Will be easier to come back to once we wanna do changes to the app internally. :)

@ECWireless lemme know if I missed any place! 🙏

### CODE REVIEW

**Review by ECWireless — APPROVED** [review]
Should all the 6s on... https://github.com/pokt-network/wpokt-app/blob/make-wpokt-decimals-constant/src/views/Stats/index.tsx ... be replaced as well? If so, I can make a quick commit right now.

**Comment by Evalir** [comment]
Ah yes! Missed those, will add them 🙏

**Comment by Evalir** [comment]
@ECWireless lemme know if that's all! If so, feel free to merge this in as soon as #33 lands :)

**Review by crisog — APPROVED** [review]

**Pull request merged**
